src/boundaries/stagger_and_step.jl

```diff
@@ -0,0 +1,217 @@
+export stagger_and_step!, stagger_trajectory!
+using LinearAlgebra:norm
+using Random
+using ProgressMeter
+
+
+"""
+    stagger_trajectory!(ds, x0, Tm, isinside; kwargs...) -> xi
+
+This function returns a point `xi` which _guarantees_ `T(xi) > 
+Tm` with a random walk search around the initial coordinates 
+`x0`. `T(xi)` is the escape time of the initial condition `xi`.  
+There is no guarantee to find such point, the parameters 
```

Comment:
I am confused here. In the first sentence you said "return a point that guarantees". Now you say there is no guarantee to find such a point...? So what would the function return if it would not find a point?

Reply:
Ah. I see below that this throws an error instead. Shouldn't error handling be done in the main function though? So that this inner function would return `nothing` instead?